docs/examples/battleship.md

```diff
@@ -0,0 +1,367 @@
+---
+sidebar_label: Battleship Game
+sidebar_position: 27
+---
+
+# Battleship Game
+
+Battleship is a strategic board game in which two players compete to sink an opponent's fleet that is hidden from their view on the playing field. Players place their ships on a grid and then take turns shooting at the squares on the opponent's field to try to find and sink his ships. The goal of the game is to be the first to destroy all of your opponent's ships.
+
+The game of Battleship requires logical thinking, strategy and luck. Players must place their ships so that they are well hidden from the enemy, with each ship having a different length, which adds variety to the tactics of the game.
+
+Battleship is a popular game, only in our case it works ** completely on the chain **. It has no backend or any centralized components, the user interface interacts directly with the smart contract uploaded in the Gear Network.
+
+Anyone can easily create their own decentralized gaming application and run it on the Gear network. To do this, we have created our own version of the game "Battleship", which you can play against a bot contract, where the winner can be determined by a tense competition. The source code is available on [GitHub](https://github.com/gear-foundation/dapps/tree/master/contracts/battleship ). This article describes the program interface, data structure, basic functions and explains their purpose. It can be used as is or modified to suit your own scenarios.
+
+## How to run
+
+Source code can be found in the [GitHub](https://github.com/gear-foundation/dapps/tree/master/contracts/battleship).
+
+Clone the repo and enter to the project directory:
+
+```shell
+git clone https://github.com/gear-foundation/dapps.git
+cd contracts/battleship
+```
+
+### ⚒️ Build program
+
+1. Install Rust on your system using the following command:
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    ```
+
+2. Add specific toolchains using these commands:
+
+    ```shell
+    rustup toolchain add nightly
+    rustup target add wasm32-unknown-unknown --toolchain nightly
+    ```
+
+3. Build the contract using the following command:
+
+    ```shell
+    cargo b -r -p "battleship*"
+    ```
+
+    You can check out our [README.md](https://github.com/gear-foundation/dapps/tree/master/contracts/battleship/README.md), which describes how to build and run contract tests.
+
+
+4. Find the program binary `battleship.opt.wasm`, `battleship_bot.opt.wasm` and `battleship.meta.txt` in `contracts/target/wasm32-unknown-unknown/release`.
+
+### 🏗️ Upload Сode
+
+1. Go to https://idea.gear-tech.io/ and choose **Vara Network Testnet** node (`wss://testnet.vara-network.io`).
+
+2. The first thing to do is ** Upload Code ** binary Wasm program file `contracts/target/wasm32-unknown-unknown/release/battleship_bot.opt.wasm`.
```

Comment:
**Bold** sentences should be without whitespaces. 
- ** Not correct **
- **Correct**
```suggestion
2. The first thing to do is **Upload Code** binary Wasm program file `contracts/target/wasm32-unknown-unknown/release/battleship_bot.opt.wasm`.
```